test/e2e-test-application/src/main.ts

```diff
@@ -1,9 +1,5 @@
 import { enableProdMode } from '@angular/core';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-// This is used for simpler testing inside dev console
-import * as LuigiClient from '@luigi-project/client';
-window['LuigiClient'] = LuigiClient;
```

Comment:
Import should work as previously - it's more about applying not compiled yet TS to window property (in result we have "undefined" value). That's why injecting "luigi-client.js" file into HTML seems to be a better choice. It can be also achieved through correct workspace configuration in Angular ('scripts' property in "angular.json" file):
https://angular.dev/reference/configs/workspace-config#extra-build-and-test-options